app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: int) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves: str) -> None:
+    for elf in elves:
+        elf.play_elf_song()
+
+
+def feast_of_the_dwarves(dwarves: int) -> None:
```

Comment:
The parameter `dwarves` is annotated as `int`, but it seems to be used as an iterable of objects with an `eat_favourite_dish` method. Consider changing the annotation to a more appropriate type, such as `list` or `Iterable`.